spec/features/users/user_reset_password_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.feature "Two Factor Authentication" do
+  describe "Password reset" do
+    let!(:user) { create(:user) }
+    xit "requires 2 factor authentication" do
```

Comment:
Yes, I don't know why it is in here because these branches should have both been off of master, not one off the other... 

I'll get it ignored again, thanks